SchibstedAccount/Manager/AppLaunchData.swift

```diff
@@ -99,4 +99,14 @@ extension AppLaunchData {
 
         return nil
     }
+
+    /**
+     Returns whether the given launch options contain a URL that is recognized as a deep link to the SchibstedAccount SDK.
+
+     - parameter launchOptions: The launch options you get through `UIApplicationDelegate.application(_:url:options:)`.
+     - parameter clientConfiguration: The current client configuration.
+     */
+    public static func doesLaunchOptionsContainRecognizedURL(_ launchOptions: [AnyHashable: Any]?, for clientConfiguration: ClientConfiguration) -> Bool {
```

Comment:
Yeah, it's exactly the same. I added it only because after writing `if let launchData = AppLaunchData(launchOptions)` I didn't find its meaning so obvious 😛